cmd/maya-apiserver/app/server/http.go

```diff
@@ -213,6 +214,9 @@ func (s *HTTPServer) registerHandlers(serviceProvider string, enableDebug bool)
 	// Request w.r.t to a single VSM entity is handled here
 	s.mux.HandleFunc("/latest/volumes/", s.wrap(latestOpenEBSVolumeRequestCounter,
 		latestOpenEBSVolumeRequestDuration, s.VSMSpecificRequest))
+
+	s.mux.HandleFunc("/latest/snapshot/", s.wrap(latestOpenEBSMetaDataRequestCounter,
```

Comment:
Should it be /latest/snapshot/ or /latest/snapshots/
What is the standard ?

Reply:
`/latest/snapshots/` is better as we have  `/latest/volumes/`

Reply:
done